extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -187,4 +187,11 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
      */
     @ConfigItem
     public boolean validateMigrationNaming;
+
+    /**
+     * Ignore migrations during validate and repair according to a given list of patterns.
+     * Patterns are comma separated.
+     */
+    @ConfigItem
+    public Optional<String[]> ignoreMigrationPatterns = Optional.empty();
```

Comment:
Not that important, but there is no need to explicitly initialize the attribute here

Reply:
If I don't initialize it here I need to check everywhere I do `flywayRuntimeConfig.ignoreMigrationPatterns.isPresent()` for nullity before and I think it defeats the purpose of using `Optional`

Reply:
Only if you create the config manually (when in unit tests for example). When injecting it this is never null

Reply:
Is it ok to leave it like this ?

Reply:
Sure, no problem